### DISCUSSION TITLE
Weekly update 6/1/2022

### DISCUSSION THREAD

**Discussion by kirubeltadesse** [Swarmies/sap-core#31] [General]
Please use this template for updating your progress!

**Comment by kirubeltadesse** [comment]
# Kirubel

###  How do you feel today?

Feeling awesome 😄 

### What did you do since the last meeting? 

- Worked on PR for #30 and #22 
- Add tickets #29  and #28 
- Started working on #27 

### What are you planning to do moving forward? 

- Creating a PR for #27 
- Updating #26 based on the PR from @dinaolmelak 

### Anything hampering your process? 

- Nothing. However, I would love to see everyone's feedback on PRs!

**Comment by dinaolmelak** [comment]
# Dinaol

###  How do you feel today?

Feeling Good 😄 

### What did you do since the last meeting? 

- Worked on PR for #30 

### What are you planning to do moving forward? 

- Work with the rest of the Team to get the #26 Done.


### Anything hampering your process? 

- Nothing really.
